src/emulator/controller.ts

```diff
@@ -365,6 +365,7 @@ export async function startAll(
     }
   }
   const hubLogger = EmulatorLogger.forEmulator(Emulators.HUB);
+  // TODO (b/217189992): Support '--only extensions` & add extensions to this logging.
```

Comment:
b/217189992 is fixed..

Reply:
Ahhhh good catch! This pr got _very_ stale, and it looks like this slipped in while merging master - I'll take another pass through and make sure nothing else is out of place